packages/terra-consumer-nav/src/components/nav-items/NavItem.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './NavItem.scss';
+import NavToggler from '../nav-toggler/NavToggler';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  // do we need these...
+  slug: PropTypes.string,
+  navType: PropTypes.oneOf(['GROUPING', 'EXTERNAL', 'APPLICATION', 'MODAL']),
+  target: PropTypes.string,
+  //
+  uri: PropTypes.string.isRequired,
+  text: PropTypes.string.isRequired,
+  isActive: PropTypes.bool,
+
+  // only those that need toggler
+  isSubNav: PropTypes.bool,
+  toggleId: PropTypes.number,
+  handleToggle: PropTypes.func,
+  isOpen: PropTypes.bool,
+  /*
+    TODO: Need to figure out what we are doing with icons or if navitems should even have icons
+
+    icon: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.element,
+    ]),
+  */
+  subNavs: PropTypes.arrayOf(PropTypes.element),
+};
+
+const defaultProps = {
+  isActive: false,
+  key: 0,
+  handleToggle: () => {},
+};
+
+const NavItem = ({
+  slug,
+  navType,
+  isSubNav,
+  target,
+  uri,
+  text,
+  isActive,
+  subNavs,
+  toggleId,
+  handleToggle,
+  isOpen,
+  ...customProps
+}) => {
+  const activeClass = cx([
+    { toggler: isSubNav },
+    { active: isActive },
+  ]);
+
+  const currentItem = (subNavs && subNavs.length > 0) ?
+    (<NavToggler
+      text={text}
+      subItems={subNavs}
+      handleToggle={open => handleToggle(toggleId, open)}
+      isOpen={isOpen}
+    />)
+  :
+  (<div className={activeClass}>
+    { isActive &&
+      <div className={cx('selection')} />
+    }
+    <a href={uri} className={cx(isSubNav ? 'sub-item' : 'nav-item')}>
```

Comment:
I think we can ditch `isSubNav` by having CSS identify if it is nested within a nav-item. 

```css
.nav-item  .nav-item {
/* sub item styles */
}
```